### PR TITLE
Youtube: unlisted videos doesn't have buttons

### LINKED ISSUE #3157: [Feature Request | Presence] YouTube
**Is your feature request related to a problem? Please describe.**
When watching an unlisted YouTube video, the button to watch the video is still present on your status - causing a private video to become public to whoever sees your status.

**Describe the solution you'd like**
The button(s) to be removed if the video is unlisted.

**Describe alternatives you've considered**
Disabling all buttons, but I would still like public videos to show.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by mlunax**
Resolves #3157
Regardless of the YouTube's language, check if video is unlisted and not assign buttons to presence.

(Random language I picked from lang list:)

### CODE REVIEW

File: websites/Y/YouTube/presence.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ presence.on("UpdateData", async () => {
     let oldYouTube: boolean = null,
       YouTubeTV: boolean = null,
       YouTubeEmbed: boolean = null,
+      UnlistedVideo = false,
```
Comment (ririxidev) [inline websites/Y/YouTube/presence.ts:89]: Change it to `unlistedVideo`. Use camelCase

**Comment by ririxidev** [comment]
no drafts.

**Comment by mlunax** [comment]
@ririxidev I wanted to lock merging cuz I found error in my code.

**Comment by mlunax** [comment]
@ririxidev How to communicate that I want to block you from merging?

**Comment by ririxidev** [comment]
Close pull request then reopen it. It should have newest commits that you will push to your fork.

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**